editor/src/components/canvas/controls/layout-parent-control.tsx

```diff
@@ -55,13 +55,14 @@ export const LayoutParentControl = React.memo((): JSX.Element | null => {
         store.editor.jsxMetadata,
         store.editor.selectedViews[0],
       )
+      const elementProps = store.editor.allElementProps[EP.toString(store.editor.selectedViews[0])]
```

Comment:
Looks like this is using the wrong element, as the above code is using the metadata for the parent element

Reply:
Good spot.